.github/workflows/integration_tests.yml

```diff
@@ -77,8 +91,8 @@ jobs:
           arguments: "--info :integration_tests:test --tests IntegrationTests.shouldMatchIosSuccessExitCodeAndPattern -Dflank-path=../test_runner/build/libs/flank.jar -Dyml-path=./src/test/resources/flank_ios.yml"
```

Comment:
This will launch the old integration test only. Once #1316 is done there will be a new Gradle task `integrationTests` that will launch all newly added cases

Reply:
So it should be updated in task #1316 
because this task is just about trigger